x-pack/plugins/ml/public/application/overview/overview_page.tsx

```diff
@@ -26,6 +27,7 @@ export const OverviewPage: FC = () => {
       <EuiPage data-test-subj="mlPageOverview">
         <EuiPageBody>
           <NodeAvailableWarning />
+          <SavedObjectsWarning />
```

Comment:
I am still seeing the callout on the Overview page, even after running Synchronize. (Testing in a non Default space).

![image](https://user-images.githubusercontent.com/7405507/102211042-e74f9380-3eca-11eb-970c-7b233ce4e6ce.png)

Reply:
Confirmed this was fixed by https://github.com/elastic/kibana/pull/85794/commits/c8fc5135e9a2e4efb8fe98dd31e3c00db342d346.